e2e/src/Gestures.ts

```diff
@@ -0,0 +1,121 @@
+import { ActionTypes } from 'webdriverio';
+
+import Utils from './Utils';
+import { ScreenPercentage } from './types';
+
+const Actions = {
```

Comment:
you created the constant but it's not used everywhere.
Also maybe worth creating one for types.